setup.py

```diff
@@ -2,8 +2,89 @@
 # -*- encoding: utf-8 -*-
 
 from setuptools import setup
+from distutils.command import build
+from distutils.command import install
+#from distutils.command import install_scripts
+import distutils
+import subprocess
+
 from kiwi.version import __version__
 
+
+class kiwi_build(build.build):
```

Comment:
This is a minor issue. I've tried `./setup.py build --help` and it gives me this output message (shortend):

```
Global options:
  [...]
Options for 'kiwi_build' command:
```

This could be confusing for some as there is no really a "kiwi_build" command. Changing it to this helps (although it reduces clarity):

```python
class build(build.build):
   # ...
```